src/Symfony/Component/Serializer/Encoder/CsvEncoder.php

```diff
@@ -21,6 +21,10 @@
 class CsvEncoder implements EncoderInterface, DecoderInterface
 {
     const FORMAT = 'csv';
+    const DELIMITER_KEY = 'csv_delimiter';
```

Comment:
@javiereguiluz Actually the `KEY` suffix is not about csv at all. It's about the constant representing the key of the option to pass in the context argument. See [DateTimeNormalizer:: FORMAT_KEY](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php#L25) for reference. 😄